### DISCUSSION TITLE
是否有工具支持taro2代码升级taro3？

### DISCUSSION THREAD

**Discussion by KehaoWu** [NervJS/taro#11550] [Q & A]
如题哈

或者有官方指南也行

**Comment by KehaoWu** [comment]
有人回应下吗

**Reply by Chen-jj** [reply]
官方没有提供升级工具，社区印象中有同学做了，可以找一找。手动升级的话可以参考文档：https://docs.taro.zone/docs/version#%E5%8D%87%E7%BA%A7%E6%8C%87%E5%8D%97

**Comment by SyMind** [comment]
https://github.com/SyMind/taro2-to-3
可以尝试下这个，帮助你将 taro 2.x 项目升级到 taro 3.x 项目，基于 jscodeshift 构建。

**Reply by KehaoWu** [reply]
谢谢，我试试
